examples/benchmark_files/create_benchmark_files.py

```diff
@@ -0,0 +1,100 @@
+import coconut
+from coconut import tools
+
+import json
+import os
+import shutil
+import subprocess
+from shutil import copy2
+from os.path import join
+
+
+"""
+This file creates benchmark pickle files used by the script 'evaluate_examples.py'.
+
+Running this file creates pickle files for the examples in the dictionary 'examples'.
+The key is the folder name of the example and the corresponding value is an int indicating the number of 
+time steps the example should be run.
+In order to exclude some examples, comment out the corresponding key-value pair in the dictionary 'examples'.
+
+To include another example, simply add a new key-value pair to the dictionary 'examples'.
+"""
+
+
+# set paths
+execute_path = os.path.realpath(os.path.dirname(__file__))
```

Comment:
execute_path and bench_mark appears to be same. Why two path variables?

Reply:
You are right.
Originally, I did not fix the decision on where to store the benchmark files and the script with respect to each other and with respect to the examples.
```
# set paths
execute_path = os.path.realpath(os.path.dirname(__file__))
tmp_path = join(execute_path, 'tmp')
examples_path = os.path.realpath(join(os.path.dirname(__file__), '..'))
bench_mark = join(examples_path, 'benchmark_files')
```
The idea is that if I wanted to move the `create_benchmark_files.py` to the examples folder, I only have to change the `examples_path` and all other paths would remain correct (`execute_path` being different from `bench_mark`).

I just kept it like this to keep the flexibility, but for the current folder structure the distinction is indeed not needed and could be removed.

Reply:
I have not changed this one. If you want, you can remove one these paths.